templates/admin/location/index.html.twig

```diff
@@ -8,6 +8,7 @@
             {% set resultcount = locations|length %}
 
             {% block buttons %}
+                <a href="{{ path("admin_location_merge") }} ">Dubbele locaties samenvoegen&nbsp;</a>
```

Comment:
This is a lot of text for the buttons, which is already a bit crowded and awkward with short button names to begin with. How about simply "Samenvoegen"?